src/kodeknekkerne/index.md

```diff
@@ -1,22 +1,8 @@
-﻿---
+---
 title: Kodeknekkerne
-external: https://kodeknekkerne.kodegenet.no/
 language: nb
+external: https://kodeknekkerne.kodegenet.no/
+tags:
+    topic: [block_based, step_based]
+    grade: [primary, secondary, junior]
 ---
-
-NRK Skole har utviklet denne serien som er en kombinasjon av gameshow og

```

Comment:
Slik sidene er idag på oppgaver.kidsakoder.no så er det lang tekst med info ved hover, samme tekst som du kan lese dersom du klikker inn på kurset og trykker på *Klikk her for mer info*. 

Fordi det er lang tekst der idag så er den ikke så nyttig (da er det bedre å gå inn på siden og lese hele teksten der). 

Eg tenker at vi burde gjøre en av to følgende:
1. Ikke ha hover i det hele tatt
2. Ha hover med kun 2-3 linjer tekst

Dersom vi går for nr 2 så bør man ha en måte å hente den infoen på. Pr nå henter den første X antall linjer fra index.md (slik eg har forstått det). Men det er ikke sikkert den teksten er tilpasset å vises isolert på denne måten. Da bør enten index.md skrives om til dette formålet også, eller at teksten hentes fra en annen fil i mappen som er laget utelukkende for hover.